spec/Scoringline/SendinblueApi/Api/SmsSpec.php

```diff
@@ -135,4 +135,10 @@ function it_should_throw_error_if_api_fails_while_send_campaign(TransformerInter
             ->duringSendCampaign(12, '+33685965788')
         ;
     }
+
+    function it_should_sanitize_phone_number()
+    {
+        $this->sanitizePhoneNumber('+1 234-324-6532')->shouldReturn('12343246532');
```

Comment:
@Nek- we don't have to pass these "00" to sendinblue. They will add them themselves.

```
+1 310-962-3135

array(3) {
  'code' =>
  string(7) "success"
  'message' =>
  string(14) "Data retrieved"
  'data' =>
  array(7) {
    'status' =>
    string(2) "OK"
    'number_sent' =>
    int(1)
    'to' =>
    string(13) "0013109623135"
    'sms_count' =>
    int(1)
    'credits_used' =>
    int(1)
    'remaining_credit' =>
    double(40249.5)
    'reference' =>
    array(1) {
      [1] =>
      string(17) "dtdjqv7qblrmvaewa"
    }
  }
}
```